### PR TITLE
New: Help text popover component and SVG

### PR DESCRIPTION
**Pull request by omgaz**
### Changes

 - Increased coverage range to 98-100% (We've kept it this high for over a year, no point letting it go)
 - Updated pre-commit hook `grep` to be more accurate
 - Fixed build/test issue of karma not showing console.log's (this was a change made in karma-mocha that stopped it working)
 - Fixed panel component title alignment issue
 - Fixed panel component title svgs being the wrong colour (design pattern = don't use black)
 - Added new help icon svg
 - Added new HelpIconPopover component and examples
 - Added bottom border for panel component titles
 - Added panel component divider support through `<hr />`
 - Added support for modals and page title `isFooter` to have supplementary information
 - Remove JSCS (it merged with ESLint)

### Screenshots

screen shot 2017-05-11 at 11 55 28
screen shot 2017-05-11 at 11 56 06
screen shot 2017-05-11 at 11 55 38

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 coverage:
   round: down
   precision: 2
-  range: 70...100
+  range: 98...100
```
Comment (mdotwills) [inline codecov.yml]: 💪

File: karma.conf.js [hunk 2]
```diff
@@ -10,13 +10,16 @@ module.exports = function configureKarma(config) {
...
+      level: 'log',
+      format: '%b %T: %m',
+      terminal: true,
+    },
     client: {
       mocha: {},
     },
     singleRun: true,
-    reporters: ['coverage', 'mocha'],
+    reporters: ['mocha', 'coverage'],
```
Comment (sharmajyo) [inline karma.conf.js]: why we changed this order?

Comment (omgaz) [inline karma.conf.js]: Haha no reason, order doesn't matter / affect anything. I think I was playing around with extra reporters.

File: src/components/alexandria/PageTitleComponent.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ PageTitleComponent.displayName = 'AlexandriaPageTitleComponent';
 PageTitleComponent.propTypes = {
   children: PropTypes.node,
   isFooter: PropTypes.bool.isRequired,
-  title: PropTypes.string,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
Comment (sonhanguyen) [inline src/components/alexandria/PageTitleComponent.jsx]: I think `PropTypes.node` suffice as it includes `string`, if we expect only one child it is `PropTypes.oneOfType([PropTypes.string, PropTypes.element])`

**Comment by mdotwills** [comment]
Merging... 🤖

**Comment by omgaz** [comment]
[(1) Adslot UI: Help component](https://trello.com/c/NG7OLcnG/6377-1-adslot-ui-help-component)

**Pull request merged**